### ISSUE TITLE
Make Enigma decompile unpicked by default

### ISSUE THREAD

**Issue by haykam821** [asiekierka/MinecraftPOMF#2867]
Currently there is a `yarnUnpicked` task in addition to the regular `yarn` task. Because unpicking is now generally stable, it should be made the default behavior.

**Comment by liach** [comment]
unpick was separate not only because it was experimental but it was slightly slower.

**Comment by Shnupbups** [comment]
I agree it should be the default, but due to the slight difference in start time perhaps the not-unpicked task should still remain. Not sure what it would be called though.

**Comment by Juuxel** [comment]
`yarnWithoutUnpick` would be fine, I guess.

**Comment by 0x000006** [comment]
Closed by #4303
